test/inductor/test_xpu_basic.py

```diff
@@ -0,0 +1,49 @@
+# Owner(s): ["module: inductor"]
+import torch
+
+try:
+    from .test_torchinductor import check_model_gpu, TestCase
+except ImportError:
+    from test_torchinductor import check_model_gpu, TestCase
+
+# TODO: Remove this file.
+# This is a temporary test case to test the base functionality of first Intel GPU Inductor integration.
+# We are working on reuse and pass the test cases in test/inductor/*  step by step.
+# Will remove this file when pass full test in test/inductor/*.
+
+
+class XpuBasicTests(TestCase):
+    common = check_model_gpu
+    device = "xpu"
+
+    def test_add(self):
+        def fn(a, b):
+            return a + b
+
+        self.common(fn, (torch.rand(2, 3, 16, 16), torch.rand(2, 3, 16, 16)))
+
+    def test_sub(self):
+        def fn(a, b):
+            return a - b
+
+        self.common(fn, (torch.rand(2, 3, 16, 16), torch.rand(2, 3, 16, 16)))
+
+    def test_mul(self):
+        def fn(a, b):
+            return a * b
+
+        self.common(fn, (torch.rand(2, 3, 16, 16), torch.rand(2, 3, 16, 16)))
+
+    def test_div(self):
+        def fn(a, b):
+            return a / b
+
+        self.common(fn, (torch.rand(2, 3, 16, 16), torch.rand(2, 3, 16, 16)))
+
+
+if __name__ == "__main__":
+    from torch._dynamo.test_case import run_tests
+    from torch.testing._internal.inductor_utils import HAS_XPU
+
+    if HAS_XPU:
+        run_tests(needs="filelock")
```

Comment:
@etaf , please check if we can reuse some Inductor test cases to verify the functionality.

Reply:
We reuse inductor in the next stacked PR.